docs/results.rst

```diff
@@ -23,6 +23,8 @@ When SRA runs are requested for download by GEMmaker, sometimes those runs fail.
 .. figure:: images/failed_runs_report.png
   :alt: Failed Runs Report
 
+There may also be a file named ``failed_runs.metadata.txt``, which is generated if metadata is not pulled from NCBI.
+
```

Comment:
As per your comment on issue #236 I think if your original concern was that the file was confusing or missing information then you might want to be a bit more verbose here to help others understand what this file is for/not for.    

Although if you do want to keep it simple, I think something like "There are files used to generate the failed run report that you may find in the results directory. These files have suffices such as 'failed_runs.metadata.txt', or ....." These can be ignored as they are used for building the failed run report"